model/src/main/java/com/flipkart/flux/domain/State.java

```diff
@@ -46,27 +62,44 @@
 
     /* Maintained by the execution engine */
     /** List of errors during state transition*/
+    @Transient
     private List<FluxError> errors;
+
     /** The Status of state transition execution*/
+    @Transient
     private Status status;
+
     /** The rollback status*/
+    @Transient
     private Status rollbackStatus;
+
     /** The number of retries attempted*/
+    @Transient
```

Comment:
State table will store only meta data, that's why I kept these as transient as these are only runtime fields.
Yes, we can take these from Audit table. (I missed rollback status in audit table, will add it)